api/src/test/java/org/openmrs/api/OrderServiceTest.java

```diff
@@ -2683,14 +2700,25 @@ public void getRevisionOrder_shouldNotReturnAVoidedRevisionOrder() {
 	 */
 	@Test
 	public void saveOrder_shouldPassForADiscontinuationOrderWithNoPreviousOrder() {
```

Comment:
I think the correct test should be one of the those in OrderIntegrationTest, possibly `shouldPlaceADrugOrder()`